app/editor/src/features/admin/reports/ReportFormDetails.tsx

```diff
@@ -54,7 +54,6 @@ export const ReportFormDetails: React.FC = () => {
         <FormikText
           name="name"
           label="Name"
-          required
```

Comment:
How do we highlight a required input without this?

Reply:
Yup handles it with the Schema, and it gives it better visual output, and messaging. (I'll add a screenshot)

Reply:
What about the initial visual output?  The user won't know which fields are required until they attempt to save?

Reply:
Yes, they will. Works exactly like the usual validation.